src/status_im/ui/screens/hardwallet/settings/views.cljs

```diff
@@ -0,0 +1,163 @@
+      [react/view {:flex-direction  :column
+                   :flex            1
+                   :align-items     :center
+                   :justify-content :space-between}
+       [components/maintain-card nil]
+       (if (zero? pin-retry-counter)
+         [pin.views/pin-view {:pin               pin
+                              :retry-counter     (when (< puk-retry-counter puk-retries) puk-retry-counter)
+                              :title-label       :t/enter-puk-code
+                              :description-label :t/enter-puk-code-description
+                              :step              step
+                              :status            status
+                              :error-label       error-label}]
+         [pin.views/pin-view {:pin               pin
+                              :retry-counter     (when (< pin-retry-counter pin-retries) pin-retry-counter)
+                              :title-label       (case step
+                                                   :current :t/current-pin
+                                                   :original :t/create-pin
+                                                   :confirmation :t/repeat-pin)
+                              :description-label (case step
+                                                   :current :t/current-pin-description
+                                                   :t/new-pin-description)
+                              :step              step
+                              :status            status
+                              :error-label       error-label}])]]]))
+
+(defn- action-row [{:keys [icon label on-press color-theme]}]
+  [react/touchable-highlight
+   {:on-press on-press}
+   [react/view {:flex-direction :row
+                :margin-top     15}
+    [react/view {:background-color (case color-theme
+                                     :red colors/red-transparent-10
+                                     colors/blue-light)
+                 :width            40
+                 :height           40
+                 :border-radius    50
+                 :align-items      :center
+                 :justify-content  :center}
+     [vector-icons/icon icon {:color (case color-theme
+                                       :red colors/red
+                                       colors/blue)}]]
+    [react/view {:align-items     :center
+                 :justify-content :center
+                 :margin-left     16}
+     [react/text {:style {:font-size 17
+                          :color     (case color-theme
+                                       :red colors/red
+                                       colors/blue)}}
+      (i18n/label label)]]]])
+
+(defn reset-card []
+  [react/view {:flex 1}
+   [status-bar/status-bar]
+   [toolbar/simple-toolbar
+    (i18n/label :t/reset-card)]
+   [react/view {:flex             1
+                :background-color :white}
+    [react/view {:margin-top  71
```

Comment:
Bunch of magic numbers. Does that work nively with multiple screen sizes?

Reply:
@jeluard it should work in theory, on my devices it looks fine.
But we will figure it out during testing on multiple devices. 
Also, those are mostly vertical margins which should be fine. Usually, the issues are with horizontal stuff, but i'm using percentages for width so should not be an issue.